src/index.js

```diff
@@ -79,6 +81,14 @@ export default function expressWs(app, httpServer, options = {}) {
     getWss: function getWss() {
       return wsServer;
     },
+    getWssClients: function getWssClients(url) {
+      if (!url) {
+        return wsServer.clients;
+      }
+
+      const clients = Array.from(wsServer.clients);
+      return clients ? clients.filter((client) => client.wsUrl === url) : [];
```

Comment:
This won't work properly for use-cases that use URL path parameters

ex: if you have something like `/chat/:id`, you would want to get all clients for a specific ID